saltysimulations/salty-747
# feat: yaw damper

* Makes the yaw damper buttons functional
* Fixes a bug where EICAS messages sometimes wouldn't appear when loading in cold and dark
* Adds the `YAW DAMPER LWR/UPR` EICAS messages

**Testing Instructions**
Load in on the runway and at a stand, make sure the yaw damper buttons are in the on position by default. Check that while the IRS is unaligned or the yaw damper buttons are off, the amber `INOP` light is illuminated on the button. Check that while the IRS is unaligned, the EICAS `YAW DAMPER LWR/UPR` messages appear.
